recipe/meta.yaml

```diff
@@ -9,8 +9,8 @@ package:
   version: {{ version }}
 
 source:
-  url: https://dist.apache.org/repos/dist/release/arrow/arrow-{{ version }}/apache-arrow-{{ version }}.tar.gz
-  sha256: 149ca6aa969ac5742f3b30d1f69a6931a533fd1db8b96712e60bf386a26dc75c
+  url: https://github.com/apache/arrow/archive/refs/tags/apache-arrow-{{ version }}.tar.gz
```

Comment:
This one should work, I cannot commit to this PR though
```suggestion
  url: http://archive.apache.org/dist/arrow/arrow-{{ version }}/apache-arrow-{{ version }}.tar.gz
```

Reply:
This one might also fix the build!

Reply:
Indeed it has. Thanks for pointing this out 😄

So are there additional files that are generated in that archive? Would we need to generate them some how ourselves when using the git repo as a source? (mostly asking for my own understanding)

Reply:
They are copied in the release archive. In the git repo they are symlinked but the symlink between the cpp and python directory is not reflected in the Github archive.

Reply:
Ah ok. So if we cloned the repo instead, it would have worked?